data/org.freedesktop.fwupd.metainfo.xml

```diff
@@ -557,7 +557,7 @@
         <p>This release fixes the following bugs:</p>
         <ul>
           <li>Be case insensitive when fixing the device model</li>
-          <li>Fix a critial warning in ccgx found by the fuzzer</li>
+          <li>Fix a critical warning in ccgx found by the fuzzer</li>
```

Comment:
Some projects don't like changing changelogs. Happy to drop this or anything else.

Reply:
It's fine for me!